### DISCUSSION TITLE
Unable to generate assets to build and debug. OmniSharp server is not running.

### DISCUSSION THREAD

**Discussion by DouglasSoares1203** [balta-io/2801#80] [Perguntas e Respostas] [answered]
Fala Pessoal, beleza?

Eu gostaria de realizar uns testes, porém na hora de depurar não aparece .Net Launch para mim...

![debug](https://user-images.githubusercontent.com/61640661/170521437-4167e2e4-beb6-4a52-a46f-0416016d4011.PNG)

Como eu posso Proceder?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @DouglasSoares1203, como vai?

Seu OmniSharp não está rodando, provavelmente deu algum problema na instalação do .NET...
Sugiro que remova o .NET/VS Code e faça as instalações novamente.
